mix.exs

```diff
@@ -27,7 +27,7 @@ defmodule BambooSes.MixProject do
     [
       {:ex_aws_ses, "~> 2.1.1"},
       {:bamboo, "~> 1.0"},
-      {:mail, "~> 0.2.0"},
+      {:mail, github: "DockYard/elixir-mail"},
```

Comment:
This could be inconvenient for other users. I'd prefer to wait for the new release of `mail`.